core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/builtin_argument/validators.go

```diff
@@ -110,3 +110,36 @@ func DurationOrNone(value starlark.Value, attributeName string) *startosis_error
 	}
 	return nil
 }
+
+func RelativeOrRemoteAbsoluteLocator(locatorStarlarkValue starlark.Value, packageId string, argNameForLogging string) *startosis_errors.InterpretationError {
+
+	locatorStarlarkStr, ok := locatorStarlarkValue.(starlark.String)
+	if !ok {
+		return startosis_errors.NewInterpretationError("Value for '%s' was expected to be a starlark.String but was '%s'", argNameForLogging, reflect.TypeOf(locatorStarlarkValue))
+	}
+	locatorStr := locatorStarlarkStr.GoString()
+
+	if isRelativeLocator(locatorStr) {
+		return nil
+	}
+
+	// if absolute and local return error
+	if isLocalLocator(locatorStr, packageId) {
+		return startosis_errors.NewInterpretationError("The locator '%s' set in attribute '%v' is not a 'local relative locator'. Local absolute locators are not allowed you should modified it to be a valid 'local relative locator'", locatorStarlarkStr, argNameForLogging)
```

Comment:
You seem to be using this function never consuming the returned error. Can't you just return a boolean then?

Reply:
I did it on this way because we are going to use it later in the argument's validator field which has to return an error